src/zjs_i2c.c

```diff
@@ -9,11 +9,14 @@
 #include "zjs_i2c.h"
```

Comment:
Update this and other files you've touched with copyright 2016-2017. Soon we'll have touched them all and be done. :)